apps/web/src/pages/v2/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { ReactNode } from "react";
```

Comment:
please use `app` directory

Reply:
Hmm, makes sense. I do think we'd have some concerns with backward compatibility if we transform the app, will investigate further.

Reply:
should be okay, pages and app directory can live side by side